policyengine_us/variables/household/income/person/general/agi_prorate_fraction_head.py

```diff
@@ -0,0 +1,28 @@
+from policyengine_us.model_api import *
+
+
+class agi_prorate_fraction_head(Variable):
```

Comment:
I don't think prorate is the right term here. If this is the head's share of AGI name it `head_share_agi` and change the entity to TaxUnit.